sysutils/pfSense-Status_Monitoring/files/usr/local/www/status_monitoring.php

```diff
@@ -123,7 +121,7 @@ function createSlug($string) {
 }
 
 //add a new view and make sure the string isn't empty
-if ($_POST['add-view'] && !empty($view_title) && ($view_title != "default")) {
+if ($_POST['add-view'] && !empty($view_title) && !preg_match("/[dD][eE][fF][aA][uU][lL][tT]/",$view_title)) {
```

Comment:
Why go through all that trouble? You could do this with:  `strtolower($view_title) != "default"` or if you really want `preg_match()`, the pattern can be `/default/i`